recipes-extended/cronie/cronie/crontab

```diff
@@ -9,7 +9,6 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 
 #m 		h 		dom		mon 	dow 	user 	command
 */10	*       *       *       *       root    /usr/sbin/logrotate /etc/logrotate.conf > /wigwag/log/cron-logrotate.log 2>&1
-0       1       *       *       *       root    /wigwag/wwrelay-utils/dev-tools/scripts/restartjob.sh >> /wigwag/log/restartjob.log 2>&1
```

Comment:
looks like this file is also explicitly installed by the recipe.  should it be removed from the image?

https://github.com/armPelionEdge/meta-pelion-edge/blob/da26ffd07b0822fa56d2a335e7c5bf36ab5a08e5/recipes-wigwag/wwrelay-utils/wwrelay-utils_1.0.1.bb#L191

Reply:
I think all those scripts will be removed when we go to pe-utils rather than continue consuming the messy edge-utils. @x85446 do we want to make that in this release or next one?